docs/recursive_systems/index.md

```diff
@@ -437,6 +451,13 @@ def split_space(curve, dir, param):
     # create the split line and convert it to a Nurbs Curve
     # (this is necessary to make the splitting work in the next function)
     split_line = rh.Line(new_pt_1, new_pt_2).ToNurbsCurve()
+    
+    #challenge2
+    split_line_length = rh.Line(new_pt_1, new_pt_2).Length
+    split_line_center = rh.Line(new_pt_1, new_pt_2).PointAtLength(split_line_length/2)
+    rotation = math.pi * rad
```

Comment:
@SaraleePam it's not clear where the `rad` variable is set. If it's based on a parameter coming from grasshopper it should probably come in as an input to this function like `dir` and `param`